flume-ng-sinks/flume-ng-kafka-sink/src/main/java/org/apache/flume/sink/kafka/KafkaSink.java

```diff
@@ -453,7 +453,9 @@ public void onCompletion(RecordMetadata metadata, Exception exception) {
 
     if (logger.isDebugEnabled()) {
       long eventElapsedTime = System.currentTimeMillis() - startTime;
-      logger.debug("Acked message partition:{} ofset:{}",  metadata.partition(), metadata.offset());
+      if (metadata != null) {
+        logger.debug("Acked message partition:{} ofset:{}", metadata.partition(), metadata.offset());
```

Comment:
This line is longer than 100 characters which brakes the checkstyle verification, could you please fix it? Thank you.